cli/azd/pkg/devcenter/provision_provider.go

```diff
@@ -446,7 +446,7 @@ func (p *ProvisionProvider) pollForEnvironment(ctx context.Context, envName stri
 			// After the resource group has been created
 			// We can start polling for a new deployment that started after we started polling
 			deployment, err := p.manager.Deployment(ctx, environment, func(d *armresources.DeploymentExtended) bool {
-				return d.Properties.Timestamp.After(pollStartTime)
+				return *d.Properties.ProvisioningState == "Running" && d.Properties.Timestamp.After(pollStartTime)
```

Comment:
Just wondering why you had to start including the Running here - What was problematic about the cases where it was in a different state?

Reply:
Just being a bit more precise here that we are picking up a running deployment.